src/hd/hd.h

```diff
@@ -19,8 +19,8 @@ extern "C" {
  */
 
 /** Interface version */
-#define HD_VERSION	21
-#define HD_MINOR_VERSION	38
+#define HD_VERSION		0
```

Comment:
can you please add note to code, that version is set automatically during install? so this is just placeholders?